python/mxnet/gluon/block.py

```diff
@@ -1052,7 +1052,7 @@ def __init__(self):
         self._callback = None
         self._monitor_all = False
         self._backend = None
-        self._backend_opts = {}
+        self._backend_opts = dict()
```

Comment:
I always had the impression that initialization with literal empty dict is preferred.

Reply:
The only exception may be function / method arguments

Reply:
will change to literal